dev-tools/ansible/roles/api-orch/defaults/main.yml

```diff
@@ -28,4 +28,14 @@ iam_server_super_admin_username: "admin"
 profile_service_host: "localhost"
 profile_service_port: 8962
 
-mariadb_connector_jar_url: "https://downloads.mariadb.com/Connectors/java/connector-java-2.0.2/mariadb-java-client-2.0.2.jar"
+mariadb_connector_jar_url: "http://downloads.mariadb.com/Connectors/java/connector-java-2.0.2/mariadb-java-client-2.0.2.jar"
```

Comment:
Why was this changed to "http"?

Reply:
MariaDB ssl certificate was expired in the day I was testing this. Now it seems like fixed. Will revert back to https now

Reply:
I again reverted back to http urls as ansible doesn't trust the ss certificates provided by mariadb

fatal: [192.168.99.102]: FAILED! => {"changed": false, "failed": true, "msg": "Failed to validate the SSL certificate for downloads.mariadb.com:443. Make sure your managed systems have a valid CA certificate installed. If the website serving the url uses SNI you need python >= 2.7.9 on your managed machine or you can install the `urllib3`, `pyopenssl`, `ndg-httpsclient`, and `pyasn1` python modules to perform SNI verification in python >= 2.6. You can use validate_certs=False if you do not need to confirm the servers identity but this is unsafe and not recommended. Paths checked for this platform: /etc/ssl/certs, /etc/pki/ca-trust/extracted/pem, /etc/pki/tls/certs, /usr/share/ca-certificates/cacert.org, /etc/ansible"}

Reply:
That's odd, I wonder why this hasn't been a problem before.  Maybe the version of Python on the VM you're testing with?  In any case, I'm fine with this change.